docs/tutorials/blob-url.md

```diff
@@ -0,0 +1,28 @@
+# Blob URL
+
+In case you need shaka-player to load a blob url mpd:
+
+#### For version v2.x
+
+- Register blob scheme to network engine (note: you don't have to do this in v3.x)
+  `shaka.net.NetworkingEngine.registerScheme('blob', shaka.net.HttpFetchPlugin);`
+- With blobs, neither extension nor MIME type can be used to deduce which manifest parser to use.
+  But there is an easy fix is register manifestParserFactory when load player
+  `player.load: player.load(uri, null, shaka.dash.DashParser);`
+- Relative URIs in the manifest are broken because the original manifest URI is lost.
+  This can be worked around with either absolute URIs or the use of BaseURL element in DASH.
```

Comment:
These comments about relative URIs are generally applicable to all versions of Shaka Player, and to HLS as well as DASH.  Would you please move them to a separate section at the top?